source/Calamari.Tests/AppServiceBehaviorFixture.cs

```diff
@@ -112,6 +112,31 @@ public async Task Deploy_WebAppZip()
             //await new AzureAppServiceBehaviour(new InMemoryLog()).Execute(runningContext);
             await AssertContent($"{site.Name}.azurewebsites.net", $"Hello {greeting}");
         }
+        
+        [Test]
+        public async Task CanDeployZip_WithSlightlyInvalidEnvironment()
+        {
+            (string packagePath, string packageName, string packageVersion) packageinfo;
```

Comment:
I feel bad about copypasting the test, but it's not sharing anything with other tests, so abstracting things away just because of some repetition feels even worse.

Reply:
If we apply the suggestion I made below, we can have two test cases - one specifically ensuring `AzureCloud` is accepted as a variable name, and one that ensures any invalid environment names throw an exception instead.